frog/imports/ui/App.jsx

```diff
@@ -34,13 +34,10 @@ export default class App extends Component {
   }
 
   switchAppByUser = () => {
-    const username = Meteor.user() ? Meteor.user().username : 'noname';
-    const app = ({
-      teacher: 'Graph Editor',
-      admin: 'Admin',
-      noname: 'Home'
-    })[username] ||
-      'Student View';
+    const username = Meteor.user() ? Meteor.user().username : null;
+    const app = username
+      ? ({ teacher: 'New Editor' })[username] || 'Graph Editor'
+      : 'Home';
```

Comment:
Sorry I think this snook in when I was rebasing from develop, will be more careful with that in the future.